libs/contact/src/effects/contact.effects.spec.ts

```diff
@@ -1,28 +1,31 @@
 import { Observable, of } from 'rxjs';
-import { DaffContactEffects } from './contact.effects';
-import { DaffContactUnion, DaffContactServiceInterface } from '@daffodil/contact';
 import { TestBed } from '@angular/core/testing';
-import { DaffTestingContactService } from 'libs/contact/testing/src/drivers/testing/contact.service';
-import { DaffContactDriver } from '../driver/injection-tokens/contact-driver.token';
 import { provideMockActions } from '@ngrx/effects/testing';
-import { DaffContactSubmit, DaffContactSuccessSubmit, DaffContactFailedSubmit, DaffContactRetry, DaffContactCancel } from '../actions/contact.actions';
 import { hot, cold } from 'jasmine-marbles';
 
+import { DaffContactSubmit, DaffContactSuccessSubmit, DaffContactFailedSubmit, DaffContactRetry, DaffContactCancel } from '../actions/contact.actions';
+import { DaffContactDriver } from '../driver/interfaces/contact-service.interface';
+import { DaffContactUnion } from '../models/contact-union';
+import { DaffContactEffects } from './contact.effects';
+
+import { DaffTestingContactService } from '../../testing/src/drivers/testing/contact.service';
+
+
 describe('DaffContactEffects', () => {
   let actions$: Observable<any>;
   let effects: DaffContactEffects<DaffContactUnion, any>;
   const mockForum = { firstName: 'John', lastName: 'Doe' };
-  let daffContactDriver: DaffContactServiceInterface<DaffContactUnion, any>;
+  let daffContactDriver;
 
   beforeEach(() => {
     TestBed.configureTestingModule({
       providers: [
+        DaffContactEffects,
+        provideMockActions(() => actions$),
         {
```

Comment:
this should be done through the DriverTestingModule or whatever it's called. Should work correctly with a relative path, right?

Reply:
So the `DaffContactTestingDriverModule` still doesn't work correctly with a relative path because the exported module is using the `DaffContactDriver` from dist instead of the one from the source code so the .spec will treat them as two separate tokens. I figured it was okay in this scenario to use the service directly because it is now imported directly from daffodil source code instead of following bad practice of using a daffodil exported service.

Reply:
I've resolved this. @lderrickable take a look at the new `tsconfig.json`.

Reply:
That's not how we'd agreed to address this. Did something change?